### PR TITLE
Add e2e tests

### PR DESCRIPTION
**Pull request by chrisface**

### CODE REVIEW

**Comment by peter-mouland** [comment]
todo:
 * [x] get toga running alongside examples app in a single command
 * [x] get e2e tests running without having to manually start a server
 * [ ] get e2e working on circleci

File: tests/config/nightwatch.conf.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/* eslint-disable */
+const argv = require('yargs')
+  .usage('Usage: $0 --target=[string]')
+  .argv;
+require('babel-core/register')({
+  only: [/app/, /tests/, /components/]
+});
+// require("babel-polyfill");
```
Comment (chrisface) [inline tests/config/nightwatch.conf.js]: Can we remove this?

**Comment by chrisface** [comment]
Looks good 👍

File: tests/config/nightwatch.conf.js (hunk 1 above)
Comment (peter-mouland) [inline tests/config/nightwatch.conf.js]: not sure which line you're talking about, but the target is useful for 'pre-prod' testing when we get to continuous deployment i.e. you set the target and the e2e tests will run a different url.

Comment (chrisface) [inline tests/config/nightwatch.conf.js]: I just meant the commented out line of code.

**Pull request merged**